tools/bumper/cnao_repo_commands_test.go

```diff
@@ -55,7 +55,10 @@ var _ = Describe("Testing internal git CNAO Repo", func() {
 
 		DescribeTable("and checking isPrAlreadyOpened function",
 			func(r isPrAlreadyOpenedParams) {
-				defer os.RemoveAll(gitCnaoRepo.gitRepo.localDir)
+				defer func(path string) {
+					Expect(os.RemoveAll(path)).To(Succeed())
```

Comment:
Not sure it is a good idea generally speaking to Expect in a defer function
it might even stop the defer calls if there were more than one
it is also "after" the test, so if there is a way to just ignore error if any it is better please
same all over please

Reply:
It's a good question..
I'm just not sure ignoring the err is a better option.
I've asked on clean-code slack, will change if they have better ideas.